### ISSUE TITLE
Groups page: fix label of the mention column

### ISSUE THREAD

**Issue by tidoust** [w3c/web-roadmaps#512]
In groups page such as the [mobile one](https://w3c.github.io/web-roadmaps/mobile/groups.html), the Mention column uses a `in %page` label. Such a label cannot be easily translated into arbitrary languages. A label such as `%feature in %page` should rather be used.

Note that this will require a second label for cases where the specification is mentioned in more than one page.

**Comment by xfq** [comment]
Yeah, `%feature in %page` should be `在%page中的%feature` (or just `%page中的%feature`) in Chinese.
